platforms-serverless-vercel/vercel-node-builder/index.js

```diff
@@ -39,7 +40,10 @@ app.get('/', async (req, res) => {
 
   // list all files in node_modules/.prisma/client
   const fs = require('fs')
-  const files = fs.readdirSync(process.env.LAMBDA_TASK_ROOT + "/node_modules/.prisma/client")
+  const generatedClientDir = path.dirname(require.resolve('.prisma/client', {
+    paths: [path.dirname(require.resolve('@prisma/client'))]
+  }))
+  const files = fs.readdirSync(generatedClientDir)
```

Comment:
This essentially helps to find the real location of the generated prisma client. Because pnpm plays with symlinks, hardlinks, and hoisting, it is able to artificially create isolated modules although they're all present in a global store. This is also why pnpm is considered a strict package manager, and that is good. This is also perfectly fine for node's resolution algorithm.

But that also means you cannot just grab the dependency at the root of your node_modules anymore, as they are strictly isolated. `.prisma/client` is always generated near `@prisma/client`, so this is what it means:
1. Look for `@prisma/client` and get that path
2.  Give that path for another search
3. Look for `.prisma/client` from `@prisma/client`'s path

Reply:
(can this be marked as resolved?)